devops/dockerfiles/Dockerfile.backend

```diff
@@ -1,25 +1,28 @@
 FROM golang:latest as backendbuilder
-WORKDIR /app
+WORKDIR /app/backend
```

Comment:
I did this on purpose because this is a template and whenever someone wants to use this template he / she can edit this to whatever version is current at that time. A fixed version would just get outdated

Reply:
sounds good to me